app/i18n/en-US/nicolive-program.json

```diff
@@ -127,80 +127,80 @@
         },
         "401": {
-          "title": "ログインしていません",
-          "message": "ログインしていないため、放送者NG設定を追加できませんでした。\nN Airメイン画面右上のログインボタンからniconicoにログインしてください。"
+          "title": "You are not logged in",
+          "message": "The block settings can not be added because you are not logged in to niconico on N Air.\nLog in to niconico from the login button on the upper right of the N Air main screen."
         },
         "403": {
-          "title": "放送者NG設定を追加できませんでした",
-          "message": "放送者NG設定を追加できませんでした。\n%{additionalMessage}"
+          "title": "The block settings could not be added",
+          "message": "The block settings could not be added\n%{additionalMessage}"
         },
         "404": {
-          "title": "番組が見つかりません",
-          "message": "番組が見つからないため、放送者NG設定を追加できませんでした。\n新規に番組を作成する場合、番組情報を更新した後、［番組作成］を押してください。"
+          "title": "There is no program",
+          "message": "You can not add the block settings because there is no program. \nIf you want to create a new program, update the program information and then press the Create New Program button."
         },
         "500": {
-          "title": "放送者NG設定を追加できませんでした",
-          "message": "放送者NG設定を追加できませんでした。\n%{additionalMessage}"
+          "title": "The block settings could not be added",
+          "message": "The Block settings could not be added\n%{additionalMessage}"
         }
       },
       "deleteFilters": {
         "network_error": {
-          "title": "ネットワークエラーが発生しました",
-          "message": "ネットワークエラーが発生したため、放送者NG設定を削除できませんでした。\nネットワーク接続が正常かどうかをご確認ください。"
+          "title": "A network error has occurred",
+          "message": "The block settings could not be deleted because of a network error.\nCheck the health of your Internet connection."
         },
         "400": {
-          "title": "放送者NG設定を削除できませんでした",
-          "message": "放送者NG設定を削除できませんでした。\n%{additionalMessage}"
+          "title": "The block settings could not be deleted",
+          "message": "The block settings could not be deleted\n%{additionalMessage}"
         },
         "401": {
-          "title": "ログインしていません",
-          "message": "ログインしていないため、放送者NG設定を削除できませんでした。\nN Airメイン画面右上のログインボタンからniconicoにログインしてください。"
+          "title": "You are not logged in",
+          "message": "The block settings can not be deleted because you are not logged in to niconico on N Air.\nLog in to niconico from the login button on the upper right of the N Air main screen."
         },
         "403": {
-          "title": "放送者NG設定を削除する権限がありません",
-          "message": "ログインしているアカウントに放送者NG設定を削除する権限がないため、番組時間を延長できませんでした。\n正しいアカウントでログインしているかを確認してください。"
+          "title": "You do not have the authority to delete the Block settings",
+          "message": "You can not load the block settings because the account you are logged into does not have authority to load the Block settings.\nMake sure you are logged in with the correct account."
```

Comment:
```suggestion
          "message": "You can not delete the block settings because the account you are logged into does not have authority to load the Block settings.\nMake sure you are logged in with the correct account."
```